src/main/java/edu/datagrok/HttpServerRunner.java

```diff
@@ -0,0 +1,8 @@
+package edu.datagrok;
+
+public class HttpServerRunner {
+    public static void main(String[] args) {
+        HttpServer httpServer = new HttpServer(8088, 10);
```

Comment:
Better to put these parameters in properties (or system variables - usually this overrides properties).